### PR TITLE
HOMS-220 Improve README and docker-compose files

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,23 +39,46 @@ The prefered way to install HOMS is to use docker
...
 
-6. Run `docker-compose`:
+6. Add environment variables to `.env` file and [Minio](https://github.com/minio/minio) credentials. Default values:
+
+  ```bash
+  HOMS_DB_PASSWORD=homs
+  HOMS_DB_NAME=homs
+  HOMS_DB_USER=homs
+  HOMS_DB_HOST=db_homs
+  BPM_DB_HOST=db_activiti
```
[the remark this answers is not in the document]
Comment (mxsmirnoff) [reply README.md]: You want to separate different kind of vars?

[the remark this answers is not in the document]
Comment (sugarfree1) [reply README.md]: Yes

**Comment by mxsmirnoff** [comment]
Updated

**Comment by mxsmirnoff** [comment]
Updated

**Comment by mxsmirnoff** [comment]
Updated

File: README.md [hunk 2]
```diff
@@ -39,23 +39,46 @@ The prefered way to install HOMS is to use docker
...
-6. Run `docker-compose`:
+6. Add bucket in Minio. Default Minio address - [http://127.0.0.1:9000](http://127.0.0.1:9000)
+  
+7. Be sure to update secret key in `config/secrets.yml`. You can generate key with this command:
+
+  ```
+  openssl rand -hex 64
+  ```
+
+9. Run `docker-compose`:
```
Comment (sugarfree1) [inline README.md]: Step 8?

File: README.md [hunk 3]
```diff
@@ -39,23 +39,46 @@ The prefered way to install HOMS is to use docker
...
+  BPM_PASSWORD=kermit
+
+  MINIO_ACCESS_KEY=minio_access_key_from_hbw_yml
+  MINIO_SECRET_KEY=minio_secret_key_from_hbw_yml
   ```
 
-6. Run `docker-compose`:
+6. Add bucket in Minio. Default Minio address - [http://127.0.0.1:9000](http://127.0.0.1:9000)
+  
+7. Be sure to update secret key in `config/secrets.yml`. You can generate key with this command:
```
Comment (sugarfree1) [inline README.md]: for Mac OS X?

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,6 +11,7 @@ v1.5.0 [unreleased]
 -   [#161](https://github.com/latera/homs/pull/161), [#6](https://github.com/latera/activiti-homs-docker/pull/6) Use activiti credentials from environment variables.
 -   [#151](https://github.com/latera/homs/pull/151) Add current version to docker container.
 -   [#151](https://github.com/latera/homs/pull/152) Remove libqtwebkit-dev.
+-   [#177](https://github.com/latera/homs/pull/177) Improved README and docker-compose files
```
Comment (sugarfree1) [inline CHANGELOG.md]: ```Improve```
Add dot at the end of line.

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**